src/main/java/uk/gov/cshr/controller/form/EmailUpdatedRecentlyEnterTokenForm.java

```diff
@@ -12,4 +13,15 @@
     private String token;
     private String domain;
     private String uid;
+    private String code;
+
+    public String toString() {
```

Comment:
Lombok @Data provides a toString method, is there a reason we need to provide our own?

Reply:
Agreed, have now replaced custom built toStrings with Lombok's version